packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx

```diff
@@ -0,0 +1,24 @@
+# Capitalization
+
+## Use sentence style caps
+
+Sentence style caps, in which only the first word of a sentence or term is capitalized, should be used across the web app.
+This makes sentences a bit more scannable and our UX copy more approachable.
+Example uses:
+
+- labels
```

Comment:
What do we mean by labels, form labels? Maybe make it specific and remove any ambiguity.